docs/SlavaUkraini/examples/accessibility/index.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: tutorial_v2
+title: A guide to basic Leaflet accessibility
+---
+
+## Accessible maps
+
+Web accessibility is the inclusive practice of ensuring no barriers exist
+that would prevent interactions or information access.
+
+This guide to Leaflet accessibility can help you create maps that are usable
+to persons of a wide range of abilities.
+
+### Preserve useful defaults
+
+Leaflet comes with a set of useful defaults.
+
+The map container and markers are keyboard operable by default,
+this enables users who are unable to use a pointing device.
+Consider the effects on your users before changing defaults such as these.
+
+### Markers must be labelled
+
+When using markers,
+it is vital to ensure each has a unique and descriptive
+[`alt`](/reference.html#marker-alt)
+or
+[`title`](/reference.html#marker-alt):
+
+<pre><code>var marker = L.marker([50.4501, 30.5234],
+  {alt: 'Kyiv'}).addTo(map) // "Kyiv" is the <a href="https://www.w3.org/TR/accname-1.1/#dfn-accessible-name"><em>accessible name</em></a> of this marker
+  .bindPopup('Kyiv, Ukraine is the birthplace of Leaflet!');</code></pre>
+
+Resulting in markers that are discernible to
+[screen reader](https://en.wikipedia.org/wiki/Screen_reader)
+users:
+
+{% include frame.html url="example.html" width=600 height=400 %}
+
+In the case of `divIcon`s,
+[custom HTML](reference.html#divicon-html)
+can otherwise provide a visual or non-visual label.
+
+### Test your maps
+
+The best way to discover accessibility issues
+is to test your maps using only a keyboard,
+as well as using a screen reader.
+You may already have a screen reader pre-installed,
+for example:
+
+- Narrator on Windows
+- TalkBack on Android
+- VoiceOver on macOS and iOS
+
+### Purely decorative maps
+
+Some maps are
+[purely decorative](https://www.w3.org/TR/WCAG21/#dfn-pure-decoration)
+and not intended for users to interact with
+(in similar fashion to background-images and -videos).
+
+Such maps should be hidden from
+<abbr title="Assistive Technologies">ATs</abbr>,
+and have no focusable descendants.
+This is to avoid the potential to confuse screen reader users,
+and to remove any unnecessary tab stops for keyboard users.
+A simple way to achieve this is to use the HTML
+[`inert` attribute](https://github.com/WICG/inert)
+polyfill:
+
+<pre><code>&lt;!-- This map is for aesthetic purposes only, and can not be interacted with! --&gt;
```

Comment:
Backticks again.

Reply:
Fixed in https://github.com/Leaflet/Leaflet/pull/8081/commits/d0c25b64d8a17d6d1566826dbf491c1ebcaf4e85, thanks.